lib/src/wallet.rs

```diff
@@ -1186,19 +1190,17 @@ impl LightWallet {
             let e = format!(
                 "Insufficient verified funds. Have {} zats, need {} zats. NOTE: funds need at least {} confirmations before they can be spent.",
                 u64::from(selected_value), u64::from(target_amount), self.transaction_context.config
-                .anchor_offset + 1
+                .reorg_buffer_offset + 1
             );
             error!("{}", e);
             return Err(e);
         }
         println!("Selected notes worth {}", u64::from(selected_value));
 
-        let orchard_anchor = self
-            .get_orchard_anchor(&orchard_notes, target_height)
-            .await?;
+        let orchard_anchor = self.get_orchard_anchor(&orchard_notes, last_height).await?;
         let mut builder = Builder::with_orchard_anchor(
             self.transaction_context.config.chain,
-            target_height,
+            submission_height,
```

Comment:
Right now, this is `reorg_buffer_offset + 1` less than what the submission height should be, which is one above the highest block known to the server.